packages/search-filter/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "search-filter",
+  "version": "0.0.1",
+  "description": "",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "react": "^16.3.1",
+    "moment": "^2.22.1",
+    "react-dom": "^16.3.3",
+    "classnames": "^2.2.6",
+    "react-input-mask": "^2.0.1",
+    "react-reactive-form": "^1.0.32"
```

Comment:
Is this absolutely necessary? I know the React devs were trying NOT to use this as a dependency.